docs/src/componentDocs/UserMenu/markdown/UserMenuAPIDocs.mdx

```diff
@@ -0,0 +1,133 @@
+| menuTitle        | Title shown when menu is open                                                                                       | `string`           | no       |                               |
+| MenuProps        | Property overrides for the MUI Menu                                                                                 | `MenuProps`        | no       |                               |
+| onClose          | Function called when the menu is closed                                                                             | `Function`         | no       |                               |
+| onOpen           | Function called when the menu is opened                                                                             | `Function`         | no       |                               |
+| useBottomSheetAt | Window pixel width at which the responsive bottom sheet menu is triggered (set to 0 to disable responsive behavior) | `number`           | no       | `theme.breakpoints.values.sm` |
+
+</Box>
+
+Any other props supplied will be provided to the root element (`div`).
+
+### Style Overrides
+
+You can override the default styles used by Brightlayer UI by:
+
+-   using the `sx` prop
+-   passing a `classes` prop with keys from the `Name` column below
+-   using the Global CSS Class in your main stylesheet
+
+For more details on styling options check out our [Styling Guide](https://github.com/brightlayer-ui/react-component-library/blob/master/docs/StyleOverridesGuide.md#style-overrides-guide).
+
+<Box>
+
+| Name        | Global CSS Class            | Description                               |
+| :---------- | :-------------------------- | :---------------------------------------- |
+| root        | .BluiUserMenu-root          | Styles applied to the root element        |
+| bottomSheet | .BluiUserMenu-bottomsheet\* | Styles applied to responsive bottom sheet |
+|             | .BluiUserMenu-avatarRoot    | Styles applied to the avatarRoot element  |
+|             | .BluiUserMenu-header        | Styles applied to the header element      |
+|             | .BluiUserMenu-headerRoot    | Styles applied to the headerRoot element  |
+|             | .BluiUserMenu-menuTitle     | Styles applied to the menuTitle element   |
+|             | .BluiUserMenu-navigation    | Styles applied to the navigation element  |
+|             | .BluiUserMenu-navGroups     | Styles applied to the navGroups element   |
+
+</Box>
+
+\* This element lives in a separate container and can't be styled via `sx` nesting rules.
+
+### User Menu Groups Object
+
+The `menuGroups` prop of the `<UserMenu>` includes many properties from the [`<DrawerNavGroup>`](https://brightlayer-ui-components.github.io/react/?path=/info/components-drawer--get-read-me-story) array found within a [`<DrawerBody>`](https://brightlayer-ui-components.github.io/react/?path=/info/components-drawer--get-read-me-story).
```

Comment:
Update this URL with dev docs api -docs. You can check how we have done for hero and channelValue.